api-beta.json

```
@@ -523,6 +693,168 @@
+        "criteria": {
+          "description": "An array of criteria to use to filter the results.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "StoreCreditTransaction": {
+      "title": "Store Credit Transaction for Customer",
+      "type": "object",
+      "description": "The store credit information for a customer transaction.",
+      "properties": {
+        "amount": {
+          "description": "The amount of the transaction.",
+          "type": "number",
+          "format": "double"
+        },
+        "type": {
+          "description": "The type of the transaction",
+          "type": "string",
+          "enum": [
+            "ISSUE",
+            "REDEMPTION",
+            "REVERSE"
+          ]
+        },
+        "notes": {
+          "description": "The notes associated with the transaction.",
+          "type": "string"
+        },
+        "user_id": {
+          "description": "The user id of the person who handled the transaction.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "sale_id": {
+          "description": "The sale id the transaction is associated with.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "client_id": {
+          "description": "The client id of the person the transaction is associated with.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "created_at": {
+          "description": "The date time the transaction was created and/or modified.",
+          "type": "string"
+        }
+      }
+    },
+    "StoreCreditTransactionParams": {
+      "title": "Store Credit Transaction Parameters for Customer",
+      "type": "object",
+      "description": "The store credit transaction parameters for a customer transaction.",
+      "properties": {
+        "store_credit_customer_id": {
+          "description": "The store credit customer id",
+          "type": "string"
+        },
+        "amount": {
+          "description": "The amount of the transaction.",
+          "type": "number",
+          "format": "double"
+        },
+        "type": {
+          "description": "The type of the transaction",
+          "type": "string",
+          "enum": [
+            "ISSUE",
+            "REDEMPTION",
+            "REVERSE"
+          ]
+        },
+        "notes": {
+          "description": "The notes associated with the transaction.",
+          "type": "string"
+        },
+        "user_id": {
+          "description": "The user id of the person who handled the transaction.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "sale_id": {
+          "description": "The sale id the transaction is associated with.",
+          "type": "string",
+          "format": "uuid"
+        },
+        "client_id": {
+          "description": "The client id of the person the transaction is associated with.",
```

Taken from gift cards:

```suggestion
          "description": "The client id is a unique transaction identifier. The client must provide this and is normally the transaction id from the client system.",
```